Source/GameEngineVisuals/Program.cs

```diff
@@ -18,7 +19,111 @@ class Program
         static char[,] g_cmMainGrid = new char[MAINGRID_X, MAINGRID_Y];
         private static Mutex g_mutex = new Mutex();
 
- 
+
+
+        public class MapVertex
+        {
+            public MapVertex(int iX, int iY)
+            {
+                m_iX = iX;
+                m_iY = iY;
+            }
+            public int m_iX;
+            public int m_iY;
+        }
+
+
+        static List<MapVertex> g_vPosition = new List<MapVertex>();                   // TODO: currently there's a bug where the last cordinate/index wont get added.
+
+  
+        //-----------------------------------------------------------------------------
+        // loadMapFromFile
+        //-----------------------------------------------------------------------------		
+        // start cordinate index from grid mid to allow for subtraction. ie (iLenght * 0,5).
+        public static void loadMapFromFile(MapVertex mapVertex)
```

Comment:
Nice to see that you guys are taking an advanced approach to how objects move in the game! 😄